StRoot/macros/bfc.C

```diff
@@ -140,7 +140,7 @@ void bfc(Int_t First, Int_t Last,
   // "-" sign before requiest means that this option is disallowed
   // Chain = "gstar" run GEANT on flight with 10 muons in range |eta| < 1 amd pT = 1GeV/c (default)
   // Dynamically link some shared libs
-  if (gClassTable->GetID("StBFChain") < 0) Load(Chain);
+  Load(Chain);
```

Comment:
What issue does this solve? The `if` statement seems a reasonable condition, doesn't it?

Reply:
Ah....I can believe that. But then to retain the same functionality, wouldn't you put something like this?

    #ifndef __CLING__
    if (gClassTable->GetID("StBFChain") < 0)
    #endif
    Load(Chain);

Reply:
Yes, but what is the underlying reason for not loading libSt_base, liblog4cxx, StStarLogger, libStChain, libStUtilities, and libStBFChain when ROOT knows about StBFChain class?